src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Use `Activity.Status` and `Activity.StatusDescription` properties instead of
+  `OpenTelemetry.Trace.Status` and `OpenTelemetry.Trace.Status.Description`
+  respectively.
+  to set activity status.
```

Comment:
Some leftovers.

Reply:
Sorry @Kielek, I don't quite get what you meant. Would you mind expanding a bit more? Thanks.

Reply:
Please check whole entry in Changelog. You should do something with ` to set activity status.`

Reply:
done!